_data/events/frankfurtAmMain.json

```diff
@@ -0,0 +1,18 @@
+{
+  "title": "GDCR 17 in Frankfurt am Main",
```

Comment:
As there is already [an event in Frankfurt](https://github.com/coderetreat/coderetreat.github.io/blob/master/_data/events/frankfurt.json), maybe it would help people if you explicitly mention *itemis AG* as the host for this one?